viz/charts/linkerd-viz/values-ha.yaml

```diff
@@ -4,6 +4,8 @@
 
 enablePodAntiAffinity: true
 
+nodeAffinity: null
```

Comment:
If we are using a string, perhaps it might be a bit better to use an empty string as opposed to null? wdyt?

Reply:
I can make it an empty string ofc. I was looking for an example, and the closest one is `tolerations` property. From what I see, [its default value is null](https://github.com/linkerd/linkerd2/blob/47105d5eb69caab6855f712516e962e431cffe5b/viz/charts/linkerd-viz/README.md?plain=1#L195).

Reply:
I guess the type of property is duck typed by `helm-docs` based on the default value. Not sure that converting it to an object changes anything.

If the value of `nodeAffinity` is not valid then `kubectl apply` will simply fail. Do you have any additional validation for `tolerations` property?

Reply:
@mateiidavid could you take a look at comments above 🙏

Reply:
Hey @michalrom089, sorry, took a while to reply. Alright, that makes sense. I'm good to keep it as is then, thank you very much for the explanation 👍🏻 . 

It would still be great to try and avoid having this hardcoded `null` value, even if the inferred value is going to be null. Think the easiest way is to just comment the field out.

```
# nodeAffinity:
```